### PR TITLE
docs: update global fee docs

### PR DESCRIPTION
**Pull request by yaruwangway**
# global fee docs update:
according to PR https://github.com/cosmos/gaia/pull/1728

update docs to remind users to insert `bypass-min-fee-msg-types` before `[telemetry]` in `app.toml`.

### CODE REVIEW

File: docs/modules/globalfee/README.md [hunk 1]
```diff
@@ -7,9 +7,10 @@ The CosmosHub has two types of fees, both of which, are the [sdk.DecCoins](https
 - min_gas_prices (are specified validator node, in the `config/app.toml` configuration file)
   
 ### Global Fees
-Global fees are set up through governance proposal which must be voted on by validators. 
+#### Global Fees Concept
+Global fees are the fees that each transaction (except [bypass fee message types](###Bypass Fees Message Types)) has to satisfy. Global fees are set up through governance proposal which must be voted on by validators.
```
Comment (mmulji-ic) [inline docs/modules/globalfee/README.md]: Perhaps change:
_Global fees are the fees that each transaction has to satisfy_ **to** Global fees are the fees that each transaction incurs

File: docs/modules/globalfee/README.md [hunk 2]
```diff
@@ -7,9 +7,10 @@ The CosmosHub has two types of fees, both of which, are the [sdk.DecCoins](https
...
   
 ### Global Fees
-Global fees are set up through governance proposal which must be voted on by validators. 
+#### Global Fees Concept
+Global fees are the fees that each transaction (except [bypass fee message types](###Bypass Fees Message Types)) has to satisfy. Global fees are set up through governance proposal which must be voted on by validators. 
 
-For a  [global fee](https://github.com/cosmos/gaia/blob/82c4353ab1b04cf656a8c95d226c30c7845f157b/x/globalfee/types/params.go#L54-L99) to be valid:
+For  [global fees](https://github.com/cosmos/gaia/blob/82c4353ab1b04cf656a8c95d226c30c7845f157b/x/globalfee/types/params.go#L54-L99) to be valid:
 - fees have to be alphabetically sorted by denomination (denom)
 - fees have to have non-negative amount, with a valid and unique denom (i.e no duplicates).
```
Comment (mmulji-ic) [inline docs/modules/globalfee/README.md]: fees have to have non-negative amount, with a valid and unique denom (i.e no duplicates).
fees **must** to have non-negative amount, with a valid and unique denom (i.e no duplicate denoms are allowed).

File: x/globalfee/README.md [hunk 3]
```diff
@@ -2,3 +2,5 @@
 
 Global fee module is copied from the [tgrade](https://github.com/confio/tgrade) respository
```
Comment (mmulji-ic) [inline x/globalfee/README.md]: The Global fee module was supplied by the great folks at [TGrade](https://github.com/confio/tgrade) 👋, with minor modifications.

File: docs/modules/globalfee/README.md (hunk 1 above)
Comment (okwme) [reply docs/modules/globalfee/README.md]:
```suggestion
Global fees are the fees that each transaction incurs—except [bypass fee message types](###Bypass Fees Message Types). Global fees are set up through a governance proposal which must be voted on by validators. 
```

**Pull request merged**